cli/config/configuration.toml.in

```diff
@@ -140,6 +140,12 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Default false
 #use_vsock = true
 
+# VFIO devices are hotplugged on a bridge by default. 
+# Enable hotplugging on root bus. This may be required for devices with
+# a large PCI bar, as this is a current limitation with hotplugging on 
+# a bridge. This value is valid for "pc" machine type.
+#hotplug_vfio_on_root_bus = true
```

Comment:
# Default false

Reply:
@devimc The comment had 
`VFIO devices are hotplugged on a bridge by default` before to explain the default. I have added the default line nonetheless.